version/1/4/ar/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+﻿الميثاق السلوكي للمشارك
+===
+عهدنا#
+
+
+من أجل تعزيز مجال مفتوح و مرحب به، نحن كمساهمون و مشرفون نتعهد بجعل المشاركة في مشروعنا و في مجموعتنا تجربة خالية من كل سلوك لا أخلاقي، بغض النظر سواء عن البنية الجسمانية، العجز، الأصل العرقي، الهوية الجنسية، الخبرة، الجنسية، المظهر الخارجي، الأصل، الدين، او الميول و الهوية الجنسية. 
+
+---
+معاييرنا#
+
+أمثلة لسلوكات تساهم في خلق جو إيجابي##
+ : 
+استعمال التحية و الغة الضمنية.*
+إحترام الآراء و التجارب المختلفة.*
+التقبل الفاضل للنقد البناء.*
+التركيز على ما هو صالح للمجموعة.*
+إظهار التعاطف تجاه أعضاء المجموعة.*
+
+أمثله لسلوكات غير مقبولة من طرف المشاركين تشمل :##
+ 
+إستعمال كلام يتضمن إيحاآت جنسية أو تصورية و كدا نظرات أو تصرفات بدافع جنسي.*
+الإستهزاء، السب أو الشتم أو النقص من التعليقات، و أيضا الهجوم الشخصي أو السياسي على باقي الأعضاء.*
+التحرش العلني أو الخاص.*
+نشر معلومات خاصة بالآخرين، مثل العنوان الشخصي أو الرقمي، بدون رخصة مصرح بها من طرف الشخص.*
+و سلوكات أخرى يمكن إعتبارها غير لائقة بإطار عملي.*
+
+---
+مسؤولياتنا#
+
+المشرفون عن المشاريع مسؤولون عن التوضيح و التصريح بمعايير السلوكات اللائقة و المقبولة، و يتوقع أن تؤخد مبادرات تأديبية، بكل عدل و إنصاف، تجاه كل سلوك غير مقبول و لاأخلاقي.
+المشرفون عن المشاريع لهم الحق في إزالة، تعديل، أو رفض التعليقات، الكودات، تعديلات ويكي، المشاكل، و كل مشاركات لا تحترم هذا الميثاق السلوكي، أو الحظر مؤقتا أو بصفة دائمة كل مشارك لسلوكات أخرى مقترفة تعتبر غير ملائمة، مستفزة، عدوانية، أو مضرة. 
+
+---
+المدى#
+
+هذا الميثاق السلوكي يطبق في المجالين، في فضاءآت المشاريع و في الفضاءآت العمومية عندما يقدم فرد ما مشروعه أو مجموعته. أمثلة تقديمات المشاريع أو المجموعات تشمل إستعمال بريدا إلكترونيا رسميا، النشر من خلال صفحة رسمية على الفايسبوك، أو التصرف كممثل معين في نشاط داخل أو خارج الإنترنيت.
+
+---
+تنفيذات#
+
+كل تصرفات فضة، تحرش، أو سلوكات أخرى غير مقبولة يمكن التبليغ عنها بالإتصال بفريق المشروع في "أضف البريد الإلكتروني". سيتم المراجعة و التحري عن كل الشكاوي، و كنتيجة ستأخد الإجراءآت اللازمة حسب الظروف. فريق المشروع مجبر أن يحافظ على سرية المراسل او المشتكي. تفاصيل أخرى عن سياسات تنفيذية محددة قد تنشر مفرقة لاحقا.

```

Comment:
@CoralineAda the changes are A LOT to be just commented here. especially RTL bullets tehy are all flipped

I'll create a newer PR for each file

I finished the first file

P.S: I'm an one of the main translators of Django and many other packages
https://www.transifex.com/user/profile/KuwaitNET/